### PR TITLE
feat: update design, nav bar and routes

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

File: src/components/layout/Navbar/NavBarLink/styles.css [hunk 1]
```diff
@@ -6,4 +6,5 @@
   font-size: 16px;
   font-family: 'Rubik-Bold';
   text-decoration: none;
+  opacity: 50%;
```
Comment (atticusofsparta) [inline src/components/layout/Navbar/NavBarLink/styles.css:9]: Should be 100% by default according to figma unless a menu is open

File: src/components/inputs/buttons/MenuButton/MenuButton.tsx [hunk 2]
```diff
@@ -5,19 +5,26 @@ import './styles.css';
...
-  icon: JSX.Element;
+  children?: JSX.Element;
 }): JSX.Element {
   return (
     <>
       <button
-        className={!show ? 'round-button' : 'round-button white-background'}
+        className={
+          !show
+            ? 'outline-button hover'
```
Comment (atticusofsparta) [inline src/components/inputs/buttons/MenuButton/MenuButton.tsx:19]: the profile button differs from the outline button: figma shows 2px border width and 6 px radius

**Comment by atticusofsparta** [comment]
Since we are updating the nav menu card we could add the icon for Log Out here. You can find the correct icon here: https://www.figma.com/file/na4spGwQ0RfniQdbh8LKJT/ar.io-Design-System?node-id=474%3A3&t=fwYbgnM05MRGX436-0
image

File: src/components/inputs/buttons/MenuButton/MenuButton.tsx (hunk 2 above)
Comment (dtfiedler) [reply src/components/inputs/buttons/MenuButton/MenuButton.tsx:19]: updated

File: src/components/layout/Navbar/NavBarLink/styles.css (hunk 1 above)
Comment (dtfiedler) [reply src/components/layout/Navbar/NavBarLink/styles.css]: turned it back to 50%, will add some logic later for when the menu is open

**Review by atticusofsparta — APPROVED** [review]

**Pull request merged**